pygsti/report/workspace.py

```diff
@@ -2461,6 +2460,9 @@ def __init__(self, ws, fn, *args):
             The arguments to `fn`.
         """
         super(WorkspacePlot, self).__init__(ws)
+        #add a plot specific option for aspect ratios
+        self.options['lock_aspect_ratio'] = True 
```

Comment:
Adding a handle for this option was the second half of the fix for the sizing issue I reported for per-sequence detail plots.